roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -34,7 +35,7 @@
     common_pod_list: "{{ galera_pod_list + ['mysqld-exporter-0'] }}"
 
 - name:  |
-    TEST Check mariadb metric endpoints
+    Check mariadb metric endpoints
```

```suggestion
    TEST Check mariadb metric endpoints
```

Revert this part of the change